snf/inject.h

```diff
@@ -0,0 +1,28 @@
+#ifndef _INJECT_H_
+#define _INJECT_H_
+
+#include <stdint.h>
+
+#ifndef USE_MOCKUP
+#include <snf.h>
+#endif
+
+static inline int
```

Comment:
you may return `struct compound_int` as in `ring_recv_many` in `wrapper.h`. Then use `intErr` to return both number of sent packets and error code if any.

```
#include "wrapper.h"
...
statc inline struct compound_int
snf_inject_send_bulk(snf_inject_t inj, int timeout_ms, int flags,
                     uintptr_t *pkts, uint32_t n_pkts, const uint32_t *lengths)
{
    struct compound_int x;
    int i;

    for (i = 0; i < n_pkts; i++)
    {
        x.rc = snf_inject_send(inj, timeout_ms, flags, (void *)pkts[i], lengths[i]);
        if (x.rc)
        {
            break;
        }
    }

    x.i[0] = i;
    return x;
}
```

Reply:
Thanks for the help. Used your recommendations.